src/Entity/Geo/Borough.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Entity\Geo;
+
+use App\Entity\EntityTimestampableTrait;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="geo_borough")
+ */
+class Borough implements ZoneableInterface
+{
+    use GeoTrait;
+    use EntityTimestampableTrait;
+
+    /**
+     * @var string[]|null
+     *
+     * @ORM\Column(type="simple_array", nullable=true)
+     */
+    private $postalCode;
+
+    /**
+     * @var int|null
+     *
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $population;
+
+    /**
+     * @var City
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\Geo\City")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $city;
+
+    public function __construct(string $code, string $name, City $city)
+    {
+        $this->code = $code;
+        $this->name = $name;
+        $this->city = $city;
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getPostalCode(): array
+    {
+        return $this->postalCode ?: [];
+    }
+
+    /**
+     * @param string[] $postalCode
+     */
+    public function setPostalCode(array $postalCode): void
+    {
+        $this->postalCode = $postalCode;
+    }
+
+    public function getPopulation(): ?int
+    {
+        return $this->population;
+    }
+
+    public function setPopulation(?int $population): void
+    {
+        $this->population = $population;
+    }
+
+    public function getCity(): City
+    {
+        return $this->city;
+    }
+
+    public function getParents(): array
+    {
+        $toMerge = [];
+
+        $toMerge[] = [$this->city];
+        $toMerge[] = $this->city->getParents();
+
+        return $toMerge ? array_values(array_unique(array_merge(...$toMerge))) : [];
```

Comment:
I think this condition is useless as it will always pass because, at least the array will look like `[ [ ], [ ] ]` at minimum

Reply:
done via a4f9789be926999b2e62e3e5b146878f7ba00edc